include/revng/Model/VerifyHelper.h

```diff
@@ -28,6 +28,7 @@ class VerifyHelper {
   std::map<const model::Type *, uint64_t> AlignmentCache;
   std::set<const model::Type *> InProgress;
   bool AssertOnFail = false;
+  mutable std::string ReasonBuffer;
```

Comment:
1. This is an hack, we should return an `llvm::Error`, but given that this part of the code will eventually also need to report *where* the error happened and report multiple errors, I'm OK with this hack for now.
   Add a TODO about this though.
2. In any case, no `mutable`

Reply:
Yeah, I've added a comment and removed `mutable`. Methods should return an `llvm::Error`, but it is not trivial to port whole project to it now - so it can be a future work.